src/backend/distributed/sql/udfs/worker_last_saved_explain_analyze/latest.sql

```diff
@@ -1,6 +1,6 @@
 
 CREATE OR REPLACE FUNCTION pg_catalog.worker_last_saved_explain_analyze()
-    RETURNS TEXT
+    RETURNS TABLE(explain_analyze_output TEXT, execution_duration DOUBLE PRECISION, extra JSONB)
```

Comment:
Do we really need the extra field to be able to add more output columns later?

Reply:
makes sense. remove it.